synapseclient/table.py

```diff
@@ -1397,20 +1397,6 @@ def from_table_query(cls, synapse, query, quoteCharacter='"', escapeCharacter="\
             header=header,
             includeRowIdAndRowVersion=includeRowIdAndRowVersion)
 
-        ## A dirty hack to find out if we got back row ID and Version
-        ## in particular, we don't get these back from aggregate queries
```

Comment:
User may ask for RowId and RowVersion but the backend may override this if the query is an aggregate query. We need to double check the backend behavior.